Apps/W1/EDocumentsConnector/app/src/Common/EDocExtCustomer.TableExt.al

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector;
+
+using Microsoft.Sales.Customer;
+
+tableextension 6360 "E-Doc. Ext. Customer" extends Customer
+{
+    fields
+    {
+        field(6360; "Service Participant Id"; Text[50])
```

Comment:
Will 50 characters always be enough? Just a thought.

Reply:
After review and specifically checking Peppol and what can be used that is country specific, 100 will better cover size. BE, as an example, could have up to 50 chars. So 100 should be more on the safe side.